### DISCUSSION TITLE
Tinycurl on Windows

### DISCUSSION THREAD

**Discussion by gvanem** [curl/curl#6875] [Q&A] [answered]
Is it possible to build / use tinycurl on Windows? (building with MSVC). 
Seems it's tucked far away in a corner in a branch on Github.

**Comment by bagder** [comment, marked as answer]
Yes, tinycurl (should) build fine on Windows.

It is also [mentioned here](https://curl.se/tiny/). With a little luck I will do another tiny-curl update soon.
